### DISCUSSION TITLE
Church of Husk reputation

### DISCUSSION THREAD

**Discussion by James-E-A** [FakeFishGames/Barotrauma#8943] [Ideas and suggestions]
Some fun ideas for the Church of Husk reputation:

* Killing [_wild_ \("natural"\) husks](https://barotraumagame.com/wiki/Husk) causes −1 rep each (maybe capped at −5 per round?);
* When undocking from an outpost, get +1 rep if there is at least 1 _more_ husked non-crew AI in the level than there was when the round started.
  * Such a specific condition to avoid unwanted interaction with future levels or events, especially modded, that might cause levels to _start_ with a husked creature
  * This bonus is increased to +5 (high impact infection) when undocking from a Military Outpost, _or_ if no living, uninfected Dweller has the Security Officer or Medic jobs
  * non-crew AI that have any % Husk Infection **are added** to the "husked" count
  * Maybe have this precipitate outpost Abandonment

If/when more in-game outcomes for faction rep are added, having these inputs to the rep system already in-place could be great.

**Comment by James-E-A** [comment]
Could also be an "alternate" victory condition for Eliminate Pirates mission: when the round ends-\(\{at least 1 Husked creature is on-board the pirate ship OR at least 1 living pirate has any% Husk Infection\} AND \{no living, uninfected pirates have the Security Officer or Medic jobs\}\)

In this case, the crew would gain the mission's reputation for Church of Husk _instead of_ the Outpost

**Comment by James-E-A** [comment]
When infecting a Research Outpost, if the +5 "total infection" bonus is NOT obtained, maybe have the outpost reputation get a random bonus in \[−10,+10\]?

There's a chance that the lab coats there do forensics and _aren't a fan_ of the parasites you let in, but also a chance that the chief scientist there is a psychopath who appreciates your contribution of a _research opportunity_…

**Reply by Hazmat616** [reply]
Or that the high ranking personnel are members of the church. One has to think about all the bureaucratic things that happened in dead space.
